bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/binding/ThingTypeProvider.java

```diff
@@ -25,23 +25,21 @@
     /**
      * Provides a collection of thing types
      *
-     * @param locale
-     *            locale (can be null)
-     *
+     * @param locale locale (can be null)
      * @return the thing types provided by the {@link ThingTypeProvider}
      */
-    Collection<ThingType> getThingTypes(Locale locale);
+    Collection<ThingType> getThingTypes(@Nullable Locale locale);
 
     /**
      * Provides a thing type for the given UID or null if no type for the
      * given UID exists.
      *
-     * @param locale
-     *            locale (can be null)
+     * @param thingTypeUID the thing type UID
+     * @param locale locale (can be null)
      * @return thing type for the given UID or null if no type for the given
      *         UID exists
      */
     @Nullable
-    ThingType getThingType(ThingTypeUID thingTypeUID, Locale locale);
+    ThingType getThingType(ThingTypeUID thingTypeUID, @Nullable Locale locale);
```

Comment:
Didn't we agree on that we only want to state if a parameter is `NonNull` and if if can be `null` we omit the `@Nullable` annotation?